software/SolarAISVN/SolarAISVN.ino

```diff
@@ -112,8 +116,8 @@ void makeIFTTTRequest() {
   Serial.println(resource);
 
   String jsonObject = String("{\"value1\":\"") + voltage[0] + "|||" + voltage[1] + "|||" + voltage[2]
-                          + "\",\"value2\":\"" + voltage[3] + "|||" + voltage[4] + "|||" + voltage[5]
-                          + "\",\"value3\":\"" + voltage[6] + "|||" + voltage[7] + "|||" + voltage[8]
+                          + "\",\"value2\":\"" + voltage[4] + "|||" + voltage[5]
+                          + "\",\"value3\":\"" + voltage[7] + "|||" + voltage[8]
```

Comment:
What happened to the submission of the third parameter?

Reply:
look at line 10. The current A and current B are measured, but we don't need the voltage values, we need the raw current. Which would just be current - and this one is submitted.